docs/api/app.md

```diff
@@ -778,7 +786,7 @@ Returns `Object`:
 **Note:** This API has no effect on
 [MAS builds][mas-builds].
 
-### `app.setLoginItemSettings(settings)` _macOS_ _Windows_
+### `app.setLoginItemSettings(settings[, path, args])` _macOS_ _Windows_
```

Comment:
These new optional arguments feel like they could just be properties of the `settings` object, any feelings towards including them there vs. separate arguments?